zebra-state/src/response.rs

```diff
@@ -0,0 +1,33 @@
+use std::sync::Arc;
+use zebra_chain::{
+    block::{self, Block},
+    transaction::Transaction,
+};
+
+// Allow *only* this unused import, so that rustdoc link resolution
+// will work with inline links.
+#[allow(unused_imports)]
+use crate::Request;
+
+#[derive(Clone, Debug, PartialEq, Eq)]
+/// A response to a state [`Request`].
+pub enum Response {
+    /// Response to [`Request::CommitBlock`] indicating that a block was
+    /// successfully committed to the state.
+    Committed(block::Hash),
+
+    /// Response to [`Request::Depth`] with the depth of the specified block.
+    Depth(Option<u32>),
+
+    /// Response to [`Request::Tip`] with the current best chain tip.
+    Tip(Option<(block::Height, block::Hash)>),
```

Comment:
This is different from the current state of #902 and I'd like to update the RFC to match this; the RFC returns a hash, but since a (height, hash) pair is available, we might as well return it.